notices.json

```diff
@@ -1,4 +1,21 @@
 [
+  {
+    "id": "gfycat_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "instanceType": "onprem",
+      "serverVersion": ["<7.8.9", "8.0.0-8.0.1", "8.1.0"]
```

Comment:
@johndavidlugtu @amyblais can you double check these server versions that the notice will be displayed:

- Servers below v7.8.9
- v8.0.0 and v8.0.1
- v8.1.0

All other versions after v8.1.0 shouldn't get the notice. Did I miss any?

Reply:
Also v7.9 and v7.10 versions should get the notice.

Reply:
Ah good catch. Missed those. I add the `"7.9-7.10"` which I'm hoping the notices is picked up by any dot releases on those versions